pkgs/tools/inputmethods/keyd/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, python39Packages, python, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "keyd";
+  version = "2.2.5-beta";
+
+  src = fetchFromGitHub {
+    owner = "rvaiya";
+    repo = "keyd";
+    rev = "v${version}";
+    sha256 = "sha256-Mb5LU8xsX8H/k3yMQiaCw4BVMphxNo25HPE0sMWTD8Y=";
+  };
+
+  makeFlags = [ "DESTDIR=${placeholder "out"}" "PREFIX=" ];
+
+  buildInputs = [ python python39Packages.xlib ];
```

Comment:
Isn't `python` in nixpkgs just Python 2? (`nix run nixpkgs#python -- --version` returns Python 2.7.18)

Reply:
Fixed in latest commit